web/packages/teleport/src/config.ts

```diff
@@ -71,6 +71,7 @@ const cfg = {
   featureLimits: {
     accessListCreateLimit: 0,
     accessMonitoringMaxReportRangeLimit: 0,
+    accessRequstMonthlyRequestLimit: 5,
```

Comment:
I thought his would come from web API?

Reply:
yes you're right, oversight, made it zero value